bin/install_requirements_linux

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+printf "\n\n====> Linux Development Setup <===="
+printf "\nEach group of packages wl be install one at a time. Follow the prompts to continue.\n"
+read -p "Press enter to continue"
+clear
+
+# Updates package lists
+sudo apt-get update
+# Updates packages to newest version
+sudo apt-get upgrade
+
+printf "\n\n====> Package upgrade complete <====\n"
+read -p "Press enter to continue"
+clear
+
+## Git
+sudo apt-get install git
+git config --global color.ui true
+git config --global user.name "your name here"
+git config --global user.email your_email_here
+
+printf "\n\n====> Git install complete <====\n"
+read -p "Press enter to continue"
+clear
+
+## Node
+curl -sL https://deb.nodesource.com/setup_6.x | sudo -E bash -
+sudo apt-get install nodejs
+sudo npm i -g webpack # installs webpack globally
```

Comment:
This should not install webpack explicitly, but install everything in the node dependencies file.